src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -76,15 +80,14 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
       "id"
     ).slice(0, size)
 
-    // TODO: get count from artworks loader to optimize pagination
-    const count = artworks.length === 0 ? 0 : MAX_ARTWORKS
+    const totalCount = filteredArtworkIds.length + backfillArtworks.length
```

Comment:
Correcting the total number of artworks.

Reply:
thanks - this was annoying